### PR TITLE
Display voice message preview before uploading

### PR DESCRIPTION
**Pull request by marcoambrosini**
preview

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/store/fileUploadStore.js [hunk 1]
```diff
@@ -211,6 +211,8 @@ const actions = {
 			let localUrl = ''
 			if (file.type === 'image/png' || file.type === 'image/gif' || file.type === 'image/jpeg') {
 				localUrl = URL.createObjectURL(file)
+			} else if (isVoiceMessage) {
```
Comment (PVince81) [inline src/store/fileUploadStore.js:214]: add this arg to the JSDoc and extend the unit test fileUploadStore.spec.js ?

File: src/components/UploadEditor.vue [hunk 2]
```diff
@@ -24,38 +24,45 @@
...
+				tag="div">
+				<template v-for="file in files">
+					<FilePreview
+						:key="file.temporaryMessage.id"
+						v-bind="file.temporaryMessage.messageParameters.file"
+						:is-upload-editor="true"
+						@remove-file="handleRemoveFileFromSelection" />
+				</template>
+				<div
+					:key="'addMore'"
```
Comment (nickvergessen) [inline src/components/UploadEditor.vue:47]: on a div? what does it do?

**Review by nickvergessen — APPROVED** [review]
Works,but see request from vincent

File: src/components/UploadEditor.vue (hunk 2 above)
Comment (marcoambrosini) [reply src/components/UploadEditor.vue:47]: It's because it's inside a loop so it needs a key

File: src/store/fileUploadStore.js (hunk 1 above)
Comment (marcoambrosini) [reply src/store/fileUploadStore.js:214]: @PVince81 I'll create an issue for the tests and I'll look into it once I have finished all the urgent tasks

File: src/components/UploadEditor.vue (hunk 2 above)
Comment (nickvergessen) [reply src/components/UploadEditor.vue:47]: but its not a component and div html elements don't have a key attribute?

**Pull request merged**